stac-spatio-temporal-asset-catalog/handout.md

```diff
@@ -1,53 +1,84 @@
-# Was ist der STAC? (TIM)
+# Was ist der STAC? (SpatioTemporal Asset Catalog)
 - einheitliches Datenformat zur Beschreibung und Katalogisierung von Geodaten
 - der STAC ist eine Spezifikation, die von den einzelnen Datenanbieter:innen umgesetzt werden muss. 
-- Nutzt "maturity classifications" um anderen Nutzer:innen zu zeigen, wie wahrscheinlich es ist, dass sich eine Erweiterung in Zukunft ändert
+- einige der öffentlich angebotenen STAC Datensätze findest du [hier](https://stacspec.org/en/about/datasets/) (z.B. Sentinel & Landsat Daten)
```

Comment:
Würde ich unten zum Minimalbeispiel packen. Das Minimalbeispiel dann auch in eine Überschrift "Weitere Ressourcen" packen. Da könnten wir auch nochmal auf unsere Beispiel Implementierung verweisen.